plugins/modules/rabbitmq_policy.py

```diff
@@ -195,7 +195,7 @@ def _policy_check(self,
         policy_name = policy_data[name_fno]
         apply_to = policy_data[apply_to_fno]
         pattern = policy_data[pattern_fno].replace('\\\\', '\\')
-        tags = json.loads(policy_data[tags_fno])
+        tags = policy_data[tags_fno]
```

Comment:
```suggestion
        try:
            tags = json.loads(policy_data[tags_fno])
        except JSONDecodeError:
            tags = policy_data[tags_fno]
```
Continuing https://github.com/ansible-collections/community.rabbitmq/pull/58#issuecomment-884683648.

Could you please apply the changes and try to run your test playbook? It should work

Also I can see that you made the changes in `main`. I would suggest using a separate branch, see the [Quick-start guide](https://github.com/ansible/community-docs/blob/main/create_pr_quick_start_guide.rst). Otherwise, if we merge the PR now, you'll probably can't continue your contribution with your fork and you'll have to remove your fork and re-fork it again.

I'm looking forward for your feedback, thank you!

Reply:
If everything is fine after applying what i'm suggesting and local testing, could you also please add a changelog fragment https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment, so users will know about the bugfix.